src/content/site/faq.md

```diff
@@ -20,6 +20,10 @@ This role recognizes people who contribute their time and knowledge to the serve
 
 ChatMod/Moderator/Senior Moderator/Admin/Owner - These people ensure that the usage of this environment reflects the guidelines as illustrated in the rules. Staff is occasionally handpicked, and sometimes there is an application process. There is a trial period for all new staff of 2-4 weeks (sometimes more, sometimes less) which comes with a trial role, however, they are treated as full staff members during this time. Please respect all staff equally; if you have an issue with a specific staff member, talk to an administrator.
 
+Senior Board Member/Board Member - This is for staff members who are currently not active moderators.
```

Comment:
Do we need these two roles to be described?

Reply:
i asked in #server-meta before making this PR

Reply:
Might be worth also adding Administrative Board Member to this list

Reply:
done 👍